smdebug/tensorflow/base_hook.py

```diff
@@ -26,16 +27,10 @@
     get_worker_id_from_tf_config,
     is_mirrored_strategy,
     is_parameter_server_strategy,
+    is_tf_version_2x,
     load_tf_config_json,
 )
 
-try:
```

Comment:
why not do this and keep supporting 1.13?
Reason for removing it?

Reply:
tf.compat.v1 and import tensorflow as tf was previously used to differentiate between 1.15 and 1.13. now, tf.compat.v1 is used by tf to differentiate 1.x and 2.x. using this try except will ensure support for 1.14+ and 2.x, but breaks 1.13. additional changes would have been required for 1.13, which have not been made based on the decision to discontinue 1.13